global-hooks/enable_cni_test.go

```diff
@@ -132,7 +132,7 @@ var _ = Describe("Global hooks :: enable_cni ::", func() {
 
 				It("Enables cni module "+module, func() {
 					Expect(f).To(ExecuteSuccessfully())
-					Expect(f.ValuesGet(module).Exists()).To(BeTrue())
+					Expect(f.ValuesGet(module).Exists()).To(BeFalse())
```

Comment:
why? You didn't change the enable_cni.go hook.

Reply:
wow, this comes with one of automatic merge of main branch 😮 
fixed